api/v1/_email/PKPEmailHandler.inc.php

```diff
@@ -137,11 +137,13 @@ public function create(ServerRequestInterface $slimRequest, APIResponse $respons
             }
         }
 
-        // Only permit emails to be sent to active users in this context
-        $params['status'] = 'active';
-        $params['contextId'] = $contextId;
 
-        $userIds = Services::get('user')->getIds($params);
+        $userIds = iterator_to_array(Repo::user()->getIds(
+            Repo::user()->getCollector()
+                ->filterByContextIds([$contextId])
+                ->filterByDisabled(false) // Only permit emails to be sent to active users in this context
```

Comment:
See https://github.com/asmecher/pkp-lib/commit/f134a8bd2b0c830bc0a01b8a69f02822d1e77aff -- I changed this to `filterByStatus` with the parameter being `Collector::STATUS_ENABLED`, `Collector::STATUS_DISABLED`, or `Collector::STATUS_ALL`:
- `STATUS_ALL` is `null` for consistency with our "null means don't care" habit. Not that we should be comparing with non-constants when they are defined for the purpose.
- The default is `STATUS_ENABLED`. There are lots of places where we didn't check the enabled status and thus returned disabled users in user lists; this tightens it up to *only* the management area. IMO we should add a filter for this over there, but I'll leave that for now.
- For now the constants are strings and null. However, once our PHP baseline supports it, I'd like to use PHP enums. This will help with autocompletion, prevent coding errors, and make error checking a built-in. Enums are due for PHP8.1: https://wiki.php.net/rfc/enumerations

I didn't go with your suggested approach of two functions because they would interact strangely with an "only enabled users" default. Presumably the default would go on the `includeDisabled` property -- meaning that calling code would need to call both `includeDisabled` and `isDisabled` if they wanted to use `isDisabled`, when you might expect just `isDisabled` to be enough.

Does that look OK? I think this will get a lot more intuitive and logical once enums are in, but this sets the stage for it.